src/tools/autocompletion/find_autocompletion_options.py

```diff
@@ -0,0 +1,45 @@
+import kdb
+import sys
+
+start_of_word = None
+
+def get_start_of_word():
+	global start_of_word
+	if (len(sys.argv) > 1):
+		start_of_word = sys.argv[1]
+
+def set_start_of_word_and_run(input_start_of_word):
+	global start_of_word
+	start_of_word = input_start_of_word
+	return find_auto_completion_options()
+
+
+def find_auto_completion_options():
```

Comment:
Maybe pass a keyset here (for easier testing)

Reply:
Do you mean the return value of find_auto_completion_options?

Reply:
I meant to pass the KeySet as parameter. So that everything with KDB is done outside. For testing you can pass manually crafted KeySets.